### ISSUE TITLE
generate random bigint for RLN Identifier

### ISSUE THREAD

**Issue by AtHeartEngineer** [Rate-Limiting-Nullifier/rlnjs#86]

**Comment by s-ekai** [comment]
@AtHeartEngineer Could you please provide more details?

**Comment by AtHeartEngineer** [comment]
> @AtHeartEngineer Could you please provide more details?

We need a utility function for someone to generate random bigints to be used as RLN Identifiers. Here are some examples:

https://github.com/Discreetly/interfaces/blob/81392484a334ffd9f778779243be220208cc5e7b/src/utils.ts#L27

https://github.com/semaphore-protocol/semaphore/blob/478e6fa38286117a56842623ef0233fba67311a3/packages/identity/src/utils.ts#L9 

We could also probably use something like this: 
https://github.com/Discreetly/interfaces/blob/81392484a334ffd9f778779243be220208cc5e7b/src/utils.ts#L6

**Comment by rutefig** [comment]
@AtHeartEngineer can I have this one? Or is @s-ekai already working on it?

**Comment by AtHeartEngineer** [comment]
I don't believe anyone is working on this right now. We (@cedoor & @vplasencia & @aguzmant103 ) did discuss creating a new version of RLN in semaphore v4 as an extension so RLN can take advantage of the semaphore v4 improvements (eddsa keys + leanIMT). So I would be hesitant to encourage you to work on this, as this version will likely be superseded by that new version.

**Comment by mahmudsudo** [comment]
can i take on this presently ?
